internal/jager-mongodb/reader_test.go

```diff
@@ -0,0 +1,110 @@
+package jager_mongodb_test
```

Comment:
can we leave the tests our of this PR until we have a framework to load test data?

Reply:
just removed it